packages/lastrev-build-prefetch-content/src/types.ts

```diff
@@ -38,22 +46,49 @@ export type WebsiteSectionPathsConfig = {
   pageContentTypes: string[];
 };
 
-export type BuildConfig = {
-  useAdapter?: boolean;
+export type FileLocationsBuildConfig = {
+  outputDirectory: string;
+  adapterConfigFile: string;
+  contentJsonDirectory: string;
+  i18nFile: string;
+  componentsDirectory: string;
+  mappingFile: string;
+  pathsFile: string;
+  settingsFile: string;
+  untranslatedPagesDirectory: string;
+  translatedPagesDirectory: string;
+  localesOutputDirectory: string;
+};
+
+export type SwitchesBuildConfig = {
+  useAdapter: boolean;
+  useWebsiteSectionPaths: boolean;
+  writeSettings: boolean;
+  writePaths: boolean;
+  writeMappings: boolean;
+  writeAdapterConfig: boolean;
+  writeLocaleData: boolean;
+  writeContentJson: boolean;
+};
+
+type OptionsBuildConfig = {
   mappings?: MappingConfig;
   paths?: Record<string, SimplePathConfig | ComplexPathConfig>;
-  websiteSectionPathsConfig: WebsiteSectionPathsConfig;
+  websiteSectionPathsConfig?: WebsiteSectionPathsConfig;
   useWebsiteSectionPaths?: boolean;
   contentPrefetch?: ContentPrefetchConfig;
   settingsInclude?: number;
   locales?: LocalesConfig;
   settingsContentType?: string;
-  writeSettings?: boolean;
```

Comment:
@prceasar if we update IF to this new version, is this something that is going to break their implementation?

Reply:
@camiloriosf No. The writeSettings field is still part of the Build Config, things just got moved around and are being composed (see line 85)